### ISSUE TITLE
how to use consistency loss without using CutMix augmentation

### ISSUE THREAD

**Issue by loseway12138** [boschresearch/unetgan#4]
Hey,there,
Awesome work, and I'm confused about how to use consistency loss without using CutMix augmentation.

**Comment by edgarschnfld** [comment]
Hi,

One can do the following two things:
(A) You can train with only CutMix consistency loss. For this, use the flag ```--consistency_loss```. 
(B) If you want to train with both CutMix consistency loss and CutMix augmentation, use instead ```--consistency_loss_and_augmentation```

So what is the technical difference between A and B? Both use the CutMix function. A imposes the consistency loss. B computes the consistency loss as well, but in addition, it uses the CutMix mask as the target for the decoder output of the discriminator (this is referred to as augmentation). The CutMix mask consists of zeros and ones, which means for some areas the discriminator target is "real" (one), while for others the target is "fake" (zero). Those values are used directly as the target for the binary cross-entropy that constitutes the non-saturating GAN loss. In contrast to B, A does not do this. A only applies the consistency loss on the logits of the discriminator predictions.

**Comment by loseway12138** [comment]
Oh,I got it,
Thank you!
